### PR TITLE
Add jquery

### PR DESCRIPTION
**Pull request by paulke43**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="pl">
+  <head>
+    <meta charset="utf-8">
+    <title>jQuery</title>
+    <link rel="stylesheet" href="css/style.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```
Comment (matwjablonski) [inline index.html]: move script to the end of document

File: js/scripts.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+//Changes css elem
```
Comment (matwjablonski) [inline js/scripts.js:2]: at first you should check that DOM is loaded

File: js/scripts.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+//Add button to each paragraphs
+var paragraphs = $('p');
+paragraphs.each(function(index, element) {
+
+    var button = '<button class="btn" data-tmp="' + index + '">Click me</button>'
+    $(element).append(button)
+
+});
+//Add event handler to each button
+$("button").click(function(){
```
Comment (matwjablonski) [inline js/scripts.js]: better use method `on` instead of `click`

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: js/scripts.js (hunk 3 above)
Comment (paulke43) [reply js/scripts.js]: I tried to use method on but it doesn't work

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**